### ISSUE TITLE
Add term fractured neck of femur

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10419]
**Preferred term label:**
Fractured neck of femur

**Synonyms**
Femoral neck fracture, broken femoral neck

**Definition (free text, please give PubMed ID)**
A partial or complete breakage of the neck of femur.

Kazley J, Bagchi K. Femoral Neck Fractures. [Updated 2023 May 8]. In: StatPearls [Internet]. Treasure Island (FL): StatPearls Publishing; 2024 Jan-. Available from: https://www-ncbi-nlm-nih-gov.mu.idm.oclc.org/books/NBK537347/

**Parent term (use hpo.jax.org/app)**
Fractures of the long bones HP:0003084
Abnormal femoral neck morphology HP:000336

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example, osteogenesis imperfecta (omim166200), Bruck syndrome 2 (omim609220)
But it can also occur after trauma (especially in the elderly)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000655
- label: Femoral neck fracture
- parents: [Femur fracture (HP:0031846)](https://hpo.jax.org/app/browse/term/HP:0031846)
- pmid: PMID:30726032
- def: A partial or complete breakage of the neck of femur.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10419
- synonyms: Fractured neck of femur(NONE)|Broken femoral neck(NONE)
